netdeploy/network.go

```diff
@@ -346,7 +346,7 @@ func (n Network) getRelayAddress(nc nodecontrol.NodeController) (relayAddress st
 	for i := 1; ; i++ {
 		relayAddress, err = nc.GetListeningAddress()
 		if err == nil {
-			return
+			return relayAddress, nil
```

Comment:
how about we convert `break` into `return "", err`, and axe the named returned values?

depends on whether Go is mart enough to let us remove the final `return`, since it becomes unreachable.